ci/build.sh

```diff
@@ -190,6 +190,8 @@ cmake \
     -DOPENVDB_USE_DEPRECATED_ABI_9=ON \
     -DOPENVDB_BUILD_VDB_PRINT=ON \
     -DOPENVDB_BUILD_VDB_LOD=ON \
+    -DOPENVDB_BUILD_VDB_TOOL=ON \
+    -DVDB_TOOL_USE_NANO=OFF \
```

Comment:
@Idclip : Hi Nick, just letting you know that have removed the vdb_tool new component in the CI and replaced it with these two lines.